request/src/main/java/com/vimeo/networking2/VimeoService.kt

```diff
@@ -211,6 +212,14 @@ internal interface VimeoService {
         @Field(SLACK_USER_PREF) slackUserPref: SlackUserPreferenceType?
     ): VimeoCall<Folder>
 
+    @FormUrlEncoded
+    @HTTP(method = "DELETE", hasBody = true)
```

Comment:
Note: passing a body in a `DELETE` is not standard, and as such is not supported by default by Retrofit. In order to properly implement it we have to use this lower level API.